tensorflow/examples/udacity/.gitignore

```diff
@@ -0,0 +1,5 @@
+notMNIST.pickle
```

Comment:
It's an intended edit :). 
It's trying to change |data_folders| to have only folders, as opposed to the relatively flimsy approach of excluding .DS_Store.